code/game/objects/items/plushes.dm

```diff
@@ -691,11 +716,95 @@
 	icon_state = "gondolaplush"
 	squeak_override = list('sound/misc/null.ogg'=1)
 
+/obj/item/toy/plush/spessgondolaplush
+	name = "astronaut gondola plushie"
+	desc = "Look at it! Go Go Astronaut Gondola!"
+	icon_state = "spessgondolaplush"
+
 /obj/item/toy/plush/flushed
 	name = "flushed plushie"
 	desc = "Hgrgrhrhg cute."
 	icon_state = "flushplush"
 
+/obj/item/choice_beacon/radial/plushie
+	name = "plushie delivery beacon"
+	desc = "Summon your new friend!"
+	icon_state = "gangtool-plushie"
+	var/static/list/plushie_list
+
+/obj/item/choice_beacon/radial/plushie/Initialize(mapload)
+	. = ..()
+	plushie_list = list(
+		/obj/item/toy/plush/bubbleplush,
+		/obj/item/toy/plush/carpplushie,
+		/obj/item/toy/plush/plushvar,
+		/obj/item/toy/plush/narplush,
```

Comment:
```suggestion
		/obj/item/toy/plush/narplush/hugbox,
```

Non Hugbox narplushies have a special interaction with Cult runes which allows them to act as an Invoker, up to twice per plush. You probably shouldn't be able to get them off spawn from a quirk.

Reply:
This request should not be dismissed - the trait should only give the cosmetic variant of the plushie if at all.